android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt

```diff
@@ -264,11 +268,25 @@ class RNPlayerViewManager(private val context: ReactApplicationContext) : Simple
                     Log.e(MODULE_NAME, "Cannot create a PlayerView, because no activity is attached.")
                     return@post
                 }
-                val playerView = PlayerView(
-                    currentActivity,
-                    player,
-                    view.config?.playerViewConfig ?: PlayerViewConfig(),
-                )
+                val rnPlayerConfigWrapper = view.config
+                    ?: playerConfig
+                        ?.let { JsonConverter.toRNPlayerViewConfigWrapper(it) }
+                val nativePlayerViewConfig = rnPlayerConfigWrapper?.playerViewConfig ?: PlayerViewConfig()
+
+                val playerView: PlayerView
+                if (nativePlayerViewConfig.uiConfig == UiConfig.Disabled ||
+                    (rnPlayerConfigWrapper?.userInterfaceType ?: UserInterfaceType.Bitmovin) ==
+                    UserInterfaceType.Bitmovin
+                ) {
+                    playerView = PlayerView(currentActivity, player, nativePlayerViewConfig)
+                } else {
+                    val overwrittenConfig = nativePlayerViewConfig.copy(uiConfig = UiConfig.Disabled)
+                    playerView = PlayerView(currentActivity, player, overwrittenConfig)
+                    val subtitleView = SubtitleView(currentActivity)
+                    subtitleView.setPlayer(player)
+                    view.addSubtitleView(subtitleView)
+                }
+
```

Comment:
I can not comment on the according line, but this code is within an else branch where we chack if the `PlayerView` was created before or not.
However, the need for the `SubtitleView` is defined by the `Player` i.e.I think we have to create/destroy the `SubtitleView` in both cases of the if branch.